pontoon/base/migrations/0040_locale_translators.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import models, migrations
+
+ACTIVITY_THRESHOLD = 10
+"""Minimal number of approved translations to promote contributor into translator."""
+
+
+def create_translators(apps, schema_editor):
+    """
+    Bug 952488 - We're assigning translation permissions to active contributors.
+    """   
+    User = apps.get_model('auth', 'User')
+
+    # We're using through migration because it's easier to map
+    # user/group primary keys without requiring to query full objects.
+    UserGroup = User.groups.through
+
+    contributors_locale = (
+        User.objects.exclude(email=settings.EXCLUDE)
```

Comment:
We use settings.EXCLUDE for excluding us from contributor lists, no need for it here.